Scenario/package.json

```diff
@@ -1,10 +1,10 @@
 {
     "name": "com.scenario.editor",
     "displayName": "Scenario Unity Plugin",
-    "version": "0.2.3",
+    "version": "0.2.5",
```

Comment:
It will still be 0.2.4 until we release everything
(PR that handle the version numebr is here: https://github.com/scenario-labs/Scenario-Unity/pull/86)

Reply:
Still 0.2.5 rather than 0.2.4